topdown/crypto.go

```diff
@@ -153,6 +211,7 @@ func init() {
 	RegisterFunctionalBuiltin1(ast.CryptoSha1.Name, builtinCryptoSha1)
 	RegisterFunctionalBuiltin1(ast.CryptoSha256.Name, builtinCryptoSha256)
 	RegisterFunctionalBuiltin1(ast.CryptoX509ParseCertificateRequest.Name, builtinCryptoX509ParseCertificateRequest)
+	RegisterFunctionalBuiltin1(ast.CryptoX509ParseRSAPrivateKey.Name, builtinCryptoX509ParseRSAPrivateKey)
```

Comment:
`RegisterFunctionalBuiltin1` is deprecated, use `RegisterBuiltinFunc` instead. See [this](https://github.com/open-policy-agent/opa/pull/3358/files) PR for reference for addig a builtin.

Reply:
Hi Ashutosh, thanks for reviewing and I was following the `CryptoX509ParseCertificateRequest` as the example to avoid "styling difference" as much as I can. However I didn't know the example was depreciated.

Now it is all refactored to use `RegisterBuiltinFunc`